Assets/Scripts/MainMenu.cs

```diff
@@ -8,8 +8,20 @@ public class MainMenu : MonoBehaviour
     // Start is called before the first frame update
     public void PlayGame()
```

Comment:
Maybe also name it PlaySingleplayerGame() for it to match the multiplayer variant?
Then, it is not ambiguous anymore, whether it is a singleplayer or a multiplayer game.